pykumo/pykumo.py

```diff
@@ -516,11 +516,12 @@ def _fetch_if_needed(self):
                                                   KUMO_RESPONSE_TIMEOUT_SECONDS))
             except Timeout as ex:
                 _LOGGER.warning("Timeout querying KumoCloud: %s", str(ex))
-            if response.ok:
-                self._kumo_dict = response.json()
-            else:
-                _LOGGER.warning("Error response from KumoCloud: %s %s}",
-                                str(response.status_code), response.text)
+            if 'response' in locals():
```

Comment:
Thanks... stylistically I'd prefer assigning `response=None` in the `except` block and testing `if response`.